xbmc/FileItem.cpp

```diff
@@ -2610,7 +2605,6 @@ void CFileItemList::StackFolders()
       // 1. rars and zips may be on slow sources? is this supposed to be allowed?
       if( !item->IsRemote()
         || item->IsSmb()
-        || item->IsNfs()
```

Comment:
this removes functionality.

Reply:
on closer inspection this can safely be removed. we do not want to encourage nfs over wan and nfs will be handled by the IsOnLAN check.